lib/shared/bucketing-assembly-script/assembly/managers/configDataManager.ts

```diff
@@ -0,0 +1,15 @@
+import { ConfigBody } from '../types'
+
+let _configData: Map<string, ConfigBody> = new Map()
```

Comment:
I wonder if we should set a size limit on this somehow? doesn't look like we can do that with a normal Map, would need to do it with an Array, which isn't great.

Reply:
I was thinking we didn't need to because it would be up to the people using the server SDK to not use too many, but I just remembered we also want to use this for CF workers.